starboard/shared/starboard/media/codec_util.cc

```diff
@@ -103,6 +103,11 @@ SbMediaAudioCodec GetAudioCodecFromString(const char* codec) {
   if (strcmp(codec, "flac") == 0) {
     return kSbMediaAudioCodecFlac;
   }
+  // For WAV, the "codecs" parameter of a MIME type refers to the WAVE format
+  // ID, where 1 represents PCM: https://datatracker.ietf.org/doc/html/rfc2361
+  if (strcmp(codec, "1") == 0) {
```

Comment:
The name of the other codecs are independent of the container, i.e. it's safe to know flac is flac, without knowing the container type.

However, to link the value "1" to PCM without knowing that the container type is WAV seems to be risky.  We may have to refine the interface of `GetAudioCodecFromString()` to make this more robust.

Reply:
Makes sense; I'll send out a follow-up PR to address this.